### PR TITLE
core localization ai --> missions

### PR DESCRIPTION
**Pull request by Bob-Murphy**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Adds 355 new strings from the following folders:
- core/functions/AI
- core/functions/Ammunition
- core/functions/Base
- core/functions/Convoy
- core/functions/Create
- core/functions/Dialogs
- core/functions/init
- core/functions/Intel
- core/functions/LTC
- core/functions/Missions

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### Notes:
- needs some adaptations to fully implement all strings
- needs proper testing
- needs check for typos and punctuation

Dev Team: Feel free to commit changes directly to this branch rather than asking me.

### CODE REVIEW

File: A3A/addons/core/Stringtable.xml [hunk 1]
```diff
@@ -5376,5 +5376,1090 @@
         <Spanish>&lt;br/&gt;&lt;t size='0.8' color='#%1' shadow='1' shadowColor='#%2' valign='top'&gt;Pulsa &lt;t color='#%3'&gt;%4&lt;/t&gt; para despejar la pista. +%5&lt;/t&gt;</Spanish>
       </Key>
     </Container>
+    <Container name="A3A_fn_ai">
+      <Key ID="STR_A3A_fn_ai_artySupport_title">
+	      <Original>Artillery Support</Original>
```
Comment (killerswin2) [inline A3A/addons/core/Stringtable.xml:5381]: bad spacing

**Review by killerswin2 — CHANGES_REQUESTED** [review]

File: A3A/addons/core/functions/LTC/fn_initLootToCrate.sqf [hunk 2]
```diff
@@ -47,7 +47,7 @@ _object addAction [
 ];
 
 _object addAction [
-    "Load loot from crate to vehicle",
+    "Load loot from crate to vehicle",//localize "STR_A3A_fn_ltc_init_addact_ltv"
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/LTC/fn_initLootToCrate.sqf:50]: done

File: A3A/addons/core/functions/LTC/fn_initLootToCrate.sqf [hunk 3]
```diff
@@ -25,13 +25,13 @@ params [["_object", objNull, [objNull]]];
...
     _params = _object actionParams _x;
-    (_params#0) isEqualTo "Load loot to crate"
+    (_params#0) isEqualTo "Load loot to crate" //TODO: Check needs to be changed to allow localization of addActions
 } != -1) exitWith {};
 
 
 //add load actions
 _object addAction [
-    "Load loot to crate",
+    "Load loot to crate", //localize "STR_A3A_fn_ltc_init_addact_ltc"
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/LTC/fn_initLootToCrate.sqf:34]: done

File: A3A/addons/core/functions/Base/fn_placementselection.sqf [hunk 4]
```diff
@@ -2,9 +2,12 @@
 FIX_LINE_NUMBERS()
 scriptName "fn_placementSelection.sqf";
 private _disabledPlayerDamage = false;
+private _titleStr = localize "STR_A3A_fn_base_placeselec_title";
 
 player allowDamage false;
 format ["%1 is Dead",name petros] hintC format ["%1 has been killed. You lost part of your assets and need to select a new HQ position far from the enemies.",name petros];
+// localize "STR_A3A_fn_base_placeselec_petros_dead" - implement into line 8
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_placementselection.sqf:9]: done

File: A3A/addons/core/functions/Base/fn_onHeadlessClientDisconnect.sqf [hunk 5]
```diff
@@ -21,3 +21,6 @@ if (_owner in hcArray) then
 		hcArray = hcArray - [_owner];
 	};
 };
+
+//STR_A3A_fn_base_hcdics_hc - needs implementation in line 14
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_onHeadlessClientDisconnect.sqf:25]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 6]
```diff
@@ -105,18 +105,18 @@ switch _typeX do
...
     };
     case "captureX":
     {
         // Uses the optional param to determine whether the call of captureX is a release or a recruit
-        _flag addAction ["<t>Release POW</t> <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa' size='1.6' shadow=2 />", { _this spawn A3A_fnc_captureX; },false,6,true,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])",4];
-        _flag addAction ["Recruit", { _this spawn A3A_fnc_captureX; },true,0,false,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])",4];
-        _flag addAction ["Interrogate", A3A_fnc_interrogate,nil,0,false,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])",4];
+        _flag addAction ["<t>Release POW</t> <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa' size='1.6' shadow=2 />", { _this spawn A3A_fnc_captureX; },false,6,true,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])",4];//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:117]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 7]
```diff
@@ -105,18 +105,18 @@ switch _typeX do
...
+        _flag addAction ["<t>Liberate</t> <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa' size='1.6' shadow=2 />", A3A_fnc_liberaterefugee,nil,6,true,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])",4]//partial string created - unsure about implementation
     };//"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa"
     case "prisonerX":
     {
-        _flag addAction ["<t>Liberate POW</t> <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa' size='1.6' shadow=2 />", A3A_fnc_liberatePOW,nil,6,true,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])",4]
+        _flag addAction ["<t>Liberate POW</t> <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa' size='1.6' shadow=2 />", A3A_fnc_liberatePOW,nil,6,true,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])",4]//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:112]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 8]
```diff
@@ -105,18 +105,18 @@ switch _typeX do
     };
     case "refugee":
     {
-        _flag addAction ["<t>Liberate</t> <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa' size='1.6' shadow=2 />", A3A_fnc_liberaterefugee,nil,6,true,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])",4]
+        _flag addAction ["<t>Liberate</t> <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa' size='1.6' shadow=2 />", A3A_fnc_liberaterefugee,nil,6,true,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])",4]//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:108]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 9]
```diff
@@ -68,18 +68,18 @@ switch _typeX do
...
             };
             //_flag addAction [format ["Revive %1",name _flag], { _this spawn A3A_fnc_actionRevive; },nil,0,false,true,"","!(_this getVariable [""helping"",false]) and (isNull attachedTo _target)"];
 
-            _actionX = _flag addAction [format ["<t>Carry %1</t> <img image='\A3\ui_f\data\igui\cfg\actions\take_ca.paa' size='1.6' shadow=2 />",name _flag], A3A_fnc_carry,nil,5,true,false,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull]) and (isNull attachedTo _target) and !(_this getVariable [""helping"",false]);",4];
-            _flag setUserActionText [_actionX,format ["Carry %1",name _flag],"<t size='2'><img image='\A3\ui_f\data\igui\cfg\actions\take_ca.paa'/></t>"];
+            _actionX = _flag addAction [format ["<t>Carry %1</t> <img image='\A3\ui_f\data\igui\cfg\actions\take_ca.paa' size='1.6' shadow=2 />",name _flag], A3A_fnc_carry,nil,5,true,false,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull]) and (isNull attachedTo _target) and !(_this getVariable [""helping"",false]);",4];//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:81]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 10]
```diff
@@ -68,18 +68,18 @@ switch _typeX do
...
+                _flag setUserActionText [_actionX,format ["Revive %1",name _flag],"<t size='2'><img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_reviveMedic_ca.paa'/></t>"];//string created, unsure about implementation
             }
             else
             {
-                _actionX = _flag addAction [format ["<t>Revive %1</t> <img size='1.8' <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_revive_ca.paa' />",name _flag], A3A_fnc_actionRevive,nil,6,true,false,"","!(_this getVariable [""helping"",false]) and (isNull attachedTo _target)",4];
-                _flag setUserActionText [_actionX,format ["Revive %1",name _flag],"<t size='2'><img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_revive_ca.paa'/></t>"];
+                _actionX = _flag addAction [format ["<t>Revive %1</t> <img size='1.8' <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_revive_ca.paa' />",name _flag], A3A_fnc_actionRevive,nil,6,true,false,"","!(_this getVariable [""helping"",false]) and (isNull attachedTo _target)",4];//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:76]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 11]
```diff
@@ -68,18 +68,18 @@ switch _typeX do
         {
             if ([_flag] call A3A_fnc_fatalWound) then
             {
-                _actionX = _flag addAction [format ["<t>Revive %1</t> <img size='1.8' <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_reviveMedic_ca.paa' />",name _flag], A3A_fnc_actionRevive,nil,6,true,false,"","!(_this getVariable [""helping"",false]) and (isNull attachedTo _target)",4];
-                _flag setUserActionText [_actionX,format ["Revive %1",name _flag],"<t size='2'><img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_reviveMedic_ca.paa'/></t>"];
+                _actionX = _flag addAction [format ["<t>Revive %1</t> <img size='1.8' <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_reviveMedic_ca.paa' />",name _flag], A3A_fnc_actionRevive,nil,6,true,false,"","!(_this getVariable [""helping"",false]) and (isNull attachedTo _target)",4];//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:71]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 12]
```diff
@@ -52,13 +52,13 @@ switch _typeX do
...
+                _flag setUserActionText [_actionX,format ["Revive %1",name _flag],"<t size='2'><img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_reviveMedic_ca.paa'/></t>"];//string created, unsure about implementation
             }
             else
             {
-                _actionX = _flag addAction [format ["<t>Revive %1 </t> <img size='1.8' <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_revive_ca.paa' />",name _flag], A3A_fnc_actionRevive,nil,6,true,true,"","!(_this getVariable [""helping"",false]) and (isNull attachedTo _target)",4];
-                _flag setUserActionText [_actionX,format ["Revive %1",name _flag],"<t size='2'><img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_revive_ca.paa'/></t>"];
+                _actionX = _flag addAction [format ["<t>Revive %1 </t> <img size='1.8' <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_revive_ca.paa' />",name _flag], A3A_fnc_actionRevive,nil,6,true,true,"","!(_this getVariable [""helping"",false]) and (isNull attachedTo _target)",4];//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:60]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 13]
```diff
@@ -52,13 +52,13 @@ switch _typeX do
         {
             if ([_flag] call A3A_fnc_fatalWound) then
             {
-                _actionX = _flag addAction [format ["<t>Revive %1 </t> <img size='1.8' <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_reviveMedic_ca.paa' />",name _flag], A3A_fnc_actionRevive,nil,6,true,true,"","!(_this getVariable [""helping"",false]) and (isNull attachedTo _target)",4];
-                _flag setUserActionText [_actionX,format ["Revive %1",name _flag],"<t size='2'><img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_reviveMedic_ca.paa'/></t>"];
+                _actionX = _flag addAction [format ["<t>Revive %1 </t> <img size='1.8' <img image='\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_reviveMedic_ca.paa' />",name _flag], A3A_fnc_actionRevive,nil,6,true,true,"","!(_this getVariable [""helping"",false]) and (isNull attachedTo _target)",4];//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:55]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 14]
```diff
@@ -12,38 +12,38 @@ switch _typeX do
...
+        petros addAction [localize "STR_A3A_fn_base_flagaction_asset_move", A3A_fnc_moveHQObject,nil,0,false,true,"","(_this == theBoss) and (petros == leader group petros)"];
 
-        petros addAction ["Build HQ here", A3A_fnc_buildHQ,nil,0,false,true,"","(_this == theBoss) and (petros != leader group petros)",4];
+        petros addAction [localize "STR_A3A_fn_base_flagaction_hq_build", A3A_fnc_buildHQ,nil,0,false,true,"","(_this == theBoss) and (petros != leader group petros)",4];
     };
     case "truckX":
     {
-        actionX = _flag addAction ["<t>Transfer Ammobox to Truck<t> <img image='\A3\ui_f\data\igui\cfg\actions\unloadVehicle_ca.paa' size='1.8' shadow=2 />", A3A_fnc_transfer,nil,6,true,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])"]
+        actionX = _flag addAction ["<t>Transfer Ammobox to Truck<t> <img image='\A3\ui_f\data\igui\cfg\actions\unloadVehicle_ca.paa' size='1.8' shadow=2 />", A3A_fnc_transfer,nil,6,true,true,"","(isPlayer _this) and (_this == _this getVariable ['owner',objNull])"]//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:46]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 15]
```diff
@@ -12,38 +12,38 @@ switch _typeX do
     case "take":
     {
         removeAllActions _flag;
-        _actionX = _flag addAction ["<t>Take the Flag<t> <img image='\A3\ui_f\data\igui\cfg\actions\takeflag_ca.paa' size='1.8' shadow=2 />", A3A_fnc_mrkWIN,nil,6,true,true,"","(isNil 'A3A_isPlayerCapturingFlag' || {!A3A_isPlayerCapturingFlag}) && (_this == _this getVariable ['owner',objNull])",4];
-        _flag setUserActionText [_actionX,"Take the Flag","<img size='2' image='\A3\ui_f\data\igui\cfg\actions\takeflag_ca.paa'/>"];
+        _actionX = _flag addAction ["<t>Take the Flag<t> <img image='\A3\ui_f\data\igui\cfg\actions\takeflag_ca.paa' size='1.8' shadow=2 />", A3A_fnc_mrkWIN,nil,6,true,true,"","(isNil 'A3A_isPlayerCapturingFlag' || {!A3A_isPlayerCapturingFlag}) && (_this == _this getVariable ['owner',objNull])",4];//partial string created - unsure about implementation
+        _flag setUserActionText [_actionX,"Take the Flag","<img size='2' image='\A3\ui_f\data\igui\cfg\actions\takeflag_ca.paa'/>"];//string created, unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:16]: done

File: A3A/addons/core/functions/Base/fn_flagaction.sqf [hunk 16]
```diff
@@ -12,38 +12,38 @@ switch _typeX do
     case "take":
     {
         removeAllActions _flag;
-        _actionX = _flag addAction ["<t>Take the Flag<t> <img image='\A3\ui_f\data\igui\cfg\actions\takeflag_ca.paa' size='1.8' shadow=2 />", A3A_fnc_mrkWIN,nil,6,true,true,"","(isNil 'A3A_isPlayerCapturingFlag' || {!A3A_isPlayerCapturingFlag}) && (_this == _this getVariable ['owner',objNull])",4];
-        _flag setUserActionText [_actionX,"Take the Flag","<img size='2' image='\A3\ui_f\data\igui\cfg\actions\takeflag_ca.paa'/>"];
+        _actionX = _flag addAction ["<t>Take the Flag<t> <img image='\A3\ui_f\data\igui\cfg\actions\takeflag_ca.paa' size='1.8' shadow=2 />", A3A_fnc_mrkWIN,nil,6,true,true,"","(isNil 'A3A_isPlayerCapturingFlag' || {!A3A_isPlayerCapturingFlag}) && (_this == _this getVariable ['owner',objNull])",4];//partial string created - unsure about implementation
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_flagaction.sqf:15]: done

File: A3A/addons/core/functions/Base/fn_commsMP.sqf [hunk 17]
```diff
@@ -55,7 +55,7 @@ if (_typeX == "tier") then
 	//playSound3D ["a3\sounds_f\sfx\beep_target.wss", player];
 	playSound "3DEN_notificationDefault";
 	//[_textX,0.8,0.5,5,0,0,2] spawn bis_fnc_dynamicText;
-	_textX = format ["War Level Changed<br/><br/>Current Level: %1",tierWar];
+	_textX = format ["War Level Changed<br/><br/>Current Level: %1",tierWar]; //localize "STR_A3A_fn_base_commsmp_warlvlchange"
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_commsMP.sqf:58]: done

File: A3A/addons/core/functions/Base/fn_canMoveHQ.sqf [hunk 18]
```diff
@@ -21,28 +21,30 @@ Example:
...
 
 if !(isNull attachedTo petros) then
 {
     if(count _result == 1) then
     {
-        ["Move HQ", "Put Petros down before you move the HQ!"] call A3A_fnc_customHint;
+        [_titleStr, localize "STR_A3A_fn_base_canmovehq_petros_down"] call A3A_fnc_customHint;
     };
-    _result pushBack "Petros currently picked up";
+    _result pushBack "Petros currently picked up";// localize "STR_A3A_fn_base_canmovehq_petros_pickedup"
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_canMoveHQ.sqf:47]: done

File: A3A/addons/core/functions/Base/fn_canMoveHQ.sqf [hunk 19]
```diff
@@ -21,28 +21,30 @@ Example:
...
 
 if ((count weaponCargo boxX >0) or (count magazineCargo boxX >0) or (count itemCargo boxX >0) or (count backpackCargo boxX >0)) then
 {
     if(count _result == 1) then
     {
-        ["Move HQ", "You must first empty your Arsenal inventory in order to move the HQ."] call A3A_fnc_customHint;
+        [_titleStr, localize "STR_A3A_fn_base_canmovehq_no_empty1"] call A3A_fnc_customHint;
     };
-    _result pushBack "Arsenal inventory must be empty";
+    _result pushBack "Arsenal inventory must be empty";// localize "STR_A3A_fn_base_canmovehq_no_empty2"
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_canMoveHQ.sqf:38]: done

File: A3A/addons/core/functions/Base/fn_calculateAggression.sqf [hunk 20]
```diff
@@ -83,7 +83,7 @@ if(_levelsChanged) then
     if(!_silent) then
     {
         //If not load progress, show message for everyone
-        _notificationText = format ["<t size='0.6' color='#C1C0BB'>Aggression level changed<br/> <t size='0.5' color='#C1C0BB'><br/>%1", _notificationText];
+        _notificationText = format ["<t size='0.6' color='#C1C0BB'>Aggression level changed<br/> <t size='0.5' color='#C1C0BB'><br/>%1", _notificationText];// localize "STR_A3A_fn_base_caclaggro_levelchange" for the "Aggression level changed" part
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_calculateAggression.sqf:86]: done

File: A3A/addons/core/functions/Base/fn_calculateAggression.sqf [hunk 21]
```diff
@@ -71,7 +71,7 @@ else
     {
         aggressionLevelInvaders = ((ceil (_newInvadersValue / 20)) min 5) max 1;
         publicVariable "aggressionLevelInvaders";
-        _notificationText = format ["%1%2 aggression level increased to %3", _notificationText, FactionGet(inv,"name"), [aggressionLevelInvaders] call A3A_fnc_getAggroLevelString];
+        _notificationText = format ["%1%2 aggression level increased to %3", _notificationText, FactionGet(inv,"name"), [aggressionLevelInvaders] call A3A_fnc_getAggroLevelString];// localize "STR_A3A_fn_base_caclaggro_inv_inc"
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_calculateAggression.sqf:74]: done

File: A3A/addons/core/functions/Base/fn_calculateAggression.sqf [hunk 22]
```diff
@@ -62,7 +62,7 @@ if(_newInvadersValue < (_levelBoundsInvaders select 0)) then
 {
     aggressionLevelInvaders = ((ceil (_newInvadersValue / 20)) min 5) max 1;
     publicVariable "aggressionLevelInvaders";
-    _notificationText = format ["%1%2 aggression level reduced to %3", _notificationText, FactionGet(inv,"name"), [aggressionLevelInvaders] call A3A_fnc_getAggroLevelString];
+    _notificationText = format ["%1%2 aggression level reduced to %3", _notificationText, FactionGet(inv,"name"), [aggressionLevelInvaders] call A3A_fnc_getAggroLevelString];// localize "STR_A3A_fn_base_caclaggro_inv_red"
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_calculateAggression.sqf:65]: done

File: A3A/addons/core/functions/Base/fn_calculateAggression.sqf [hunk 23]
```diff
@@ -53,7 +53,7 @@ else
     {
         aggressionLevelOccupants = ((ceil (_newOccupantsValue / 20)) min 5) max 1;
         publicVariable "aggressionLevelOccupants";
-        _notificationText = format ["%1 aggression level increased to %2<br/>", FactionGet(occ,"name"), [aggressionLevelOccupants] call A3A_fnc_getAggroLevelString];
+        _notificationText = format ["%1 aggression level increased to %2<br/>", FactionGet(occ,"name"), [aggressionLevelOccupants] call A3A_fnc_getAggroLevelString];// localize "STR_A3A_fn_base_caclaggro_occ_inc"
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_calculateAggression.sqf:56]: done

File: A3A/addons/core/functions/Base/fn_calculateAggression.sqf [hunk 24]
```diff
@@ -44,7 +44,7 @@ if(_newOccupantsValue < (_levelBoundsOccupants select 0)) then
 {
     aggressionLevelOccupants = ((ceil (_newOccupantsValue / 20)) min 5) max 1;
     publicVariable "aggressionLevelOccupants";
-    _notificationText = format ["%1 aggression level reduced to %2<br/>", FactionGet(occ,"name"), [aggressionLevelOccupants] call A3A_fnc_getAggroLevelString];
+    _notificationText = format ["%1 aggression level reduced to %2<br/>", FactionGet(occ,"name"), [aggressionLevelOccupants] call A3A_fnc_getAggroLevelString];// localize "STR_A3A_fn_base_caclaggro_occ_red"
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_calculateAggression.sqf:47]: done

File: A3A/addons/core/functions/Base/fn_airspaceControl.sqf [hunk 25]
```diff
@@ -169,8 +169,8 @@ while {!(isNull _vehicle) && {alive _vehicle && {count (crew _vehicle) != 0}}} d
 
         {
             //Assuming you only get a single one each second, need to split it otherwise
-            private _warningText = format ["Unidentified helicopter<br/><br/>You are closing in on the airspace of %1.<br/><br/> Change your course or we will take defensive actions!", [_x] call A3A_fnc_localizar];
-            ["Undercover", _warningText] remoteExec ["A3A_fnc_customHint", (crew _vehicle)];
+            private _warningText = format ["Unidentified helicopter<br/><br/>You are closing in on the airspace of %1.<br/><br/> Change your course or we will take defensive actions!", [_x] call A3A_fnc_localizar]; //I don't get this weird line but here is the string - STR_A3A_fn_base_airspacecontrol_warning
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply A3A/addons/core/functions/Base/fn_airspaceControl.sqf:172]: done

File: A3A/addons/core/Stringtable.xml (hunk 1 above)
Comment (Bob-Murphy) [reply A3A/addons/core/Stringtable.xml:5381]: done

**Review by killerswin2 — APPROVED** [review]

**Pull request merged**